src/components/patch-details/patch-details.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Link as RouterLink } from "react-router-dom";
+
+import Cookies from "js-cookie";
+
+import { locale } from "../../services";
+
+import { Asset } from "@mtfh/common/lib/api/asset/v1";
+import { Patch, usePatchOrArea } from "@mtfh/common/lib/api/patch/v1";
+import {
+  Button,
+  Heading,
+  SummaryList,
+  SummaryListItem,
+  Text,
+} from "@mtfh/common/lib/components";
+
+const PatchTable = ({ patches }: { patches: Patch[] }) => {
+  const assetPatch = patches.find((patch) => patch.patchType === "patch");
+  const parentId = assetPatch?.parentId || "";
+  const parentAreaReq = usePatchOrArea(parentId);
+  const parentArea = parentAreaReq.data;
+  if (!assetPatch) return <h1>Error: Not found</h1>;
+
+  const housingOfficerName = assetPatch?.responsibleEntities[0]?.name;
+
+  const { patchLabel, housingOfficerLabel, areaManagerLabel } = locale.patchDetails;
+  return (
+    <SummaryList overrides={[2 / 3]}>
+      <SummaryListItem title={patchLabel} data-testid="patch-name" key="P">
```

Comment:
The key should be something like "patchName",  I think it might be optional as well

Reply:
Renamed the keys in 0b3eb9f261ec90209a18ea97ed0548056a737ce4